benchmark/launch_software.sh

```diff
@@ -29,196 +29,123 @@ species_tree_fp=$3
-    TIME="$( time ($jane_install_dir/jane-cli.sh $input_file_nex 1>$output_file 2>$stderr) 2>&1)"
-    python ${scripts_dir}/parse_output.py --hgt-results-fp $output_file --method 'jane4' >> $hgt_summary_file
-    printf "\t" >> $hgt_summary_file
-    user_time=$(echo $TIME | awk '{print $1;}')
-    wall_time=$(echo $TIME | awk '{print $2;}')
-    total_user_time_jane=$(echo $total_user_time_jane + $user_time | bc)
-    total_wall_time_jane=$(echo $total_wall_time_jane + $wall_time | bc)
-    rm $output_file
-
-    # CONSEL (AU Test)
-    # input conditions: matrix of the site-wise log-likelihoods
-    # (a) if no MSA provided, CLUSTALW (align sequences)
-    # (b) if MSA provided (ex. ALF), Fasta2Phylip.py
-    # TREE-PUZZLE (reconstruct phylogenetic tree using maximum likelihood)
-    # CONSEL (apply AU Test on matrix)
-    if [ "$verbose" == "true" ]
-    then
-        echo -e "\tTree-Puzzle and CONSEL"
-    fi
-    gene_msa_fasta_fp=$gene_msa_dir/"MSA_${gene_number}_aa.fa"
-    gene_msa_phylip_fp=$working_dir/"MSA_${gene_number}_aa.phy"
-    python ${scripts_dir}/reformat_input.py --method 'tree-puzzle' \
-                                            --gene-tree-fp $gene_tree \
-                                            --species-tree-fp $species_tree_fp \
-                                            --gene-msa-fa-fp $gene_msa_fasta_fp \
-                                            --output-tree-fp $input_file_nwk \
-                                            --output-msa-phy-fp $gene_msa_phylip_fp
-    puzzle -wsl $gene_msa_phylip_fp $input_file_nwk < $working_dir/puzzle_cmd.txt 1>$stdout 2>$stderr
-    # makermt removes the .sitelh extension and writes to the edited file path
-    # which would overwrite the Newick tree. Rename the input file to avoid this.
-    mv ${input_file_nwk}.sitelh ${input_file_nwk}_puzzle.sitelh
-    TIME="$( time (makermt --puzzle ${input_file_nwk}_puzzle.sitelh 1>$stdout 2>$stderr) 2>&1)"
-    consel ${input_file_nwk}_puzzle 1>$stdout 2>$stderr
-    catpv ${input_file_nwk}_puzzle.pv 1>$output_file 2>$stderr
-    python ${scripts_dir}/parse_output.py --hgt-results-fp $output_file --method 'consel' >> $hgt_summary_file
-    printf "\n" >> $hgt_summary_file
-    user_time=$(echo $TIME | awk '{print $1;}')
-    wall_time=$(echo $TIME | awk '{print $2;}')
-    total_user_time_consel=$(echo $total_user_time_consel + $user_time | bc)
-    total_wall_time_consel=$(echo $total_wall_time_consel + $wall_time | bc)
-
-    ## Clean up
-    rm $output_file
-    rm $gene_msa_phylip_fp
-    rm ${input_file_nwk}_puzzle.pv
-    rm ${input_file_nwk}_puzzle.sitelh
-    i=$((i+1))
-done
-
-echo "Total time T-REX: $total_user_time_trex"
-echo "Total time RANGER-DTL: $total_user_time_rangerdtl"
-echo "Total time RIATA-HGT: $total_user_time_riatahgt"
-echo "Total time Jane 4: $total_user_time_jane"
-echo "Total time AU-Test: $total_user_time_consel"
+## run T-REX
+echo "source ~/.bash_profile; \
```

Comment:
This is only required for barnacle, right? Should it be something defined at the top of the file and appended to all commands?